src/machine/behavior/is/arithmetic/addu.rs

```diff
@@ -1,6 +1,11 @@
 use machine::state::State;
 
-pub fn addu(_state: &mut State, _x: u8, _y: u8, _z: u8) {
-    unimplemented!();
+pub fn addu(state: &mut State, x: u8, y: u8, z: u8) {
+    //  load operands
+    let op1: u64 = state.gpr[y].into();
+    let op2: u64 = state.gpr[z].into();
+    // execute
```

Comment:
I will change most of them along with other corrections and I won't make a special commit for it. I hope this is ok.